### DISCUSSION TITLE
Battery Charge Icon - Not Properly Refreshed

### DISCUSSION THREAD

**Discussion by FileCity** [nutdotnet/WinNUT-Client#187] [Q&A]
I don't know if it's a reported bug but the battery charge icon is not refreshing when my UPS status changed from "OB DISCHRG" to "OL CHARG". It always display the discharging icon next to the gauge even if the UPS status variable is OK and charging.

**Comment by gbakeman** [comment]
It sounds likely that you've found a new bug - getting all the icons and statuses updating correctly has been tricky. I'll look into this more and convert to an issue when I can confirm. Thanks for posting this!
